Add comments on ImageMagick requirement (#10386)
sphinx-doc/sphinx: doc/usage/extensions/imgconverter.rst

builders.  For example, it allows you to use SVG images with LaTeX builder.
As a result, you don't mind what image format the builder supports.

> Internally, this extension uses Imagemagick_ to convert images,
> you'll have to install it on your machine.

Reviewer: ```suggestion
The extension uses ImageMagick_ to perform conversions, 
and will fail if ImageMagick is not installed.
```

> .. _Imagemagick: https://www.imagemagick.org/script/index.php

Reviewer: ```suggestion
.. _ImageMagick: https://www.imagemagick.org
```

.. confval:: image_converter

   A path to a conversion command.  By default, the imgconverter finds
   the command from search paths.

>    On Unix platform, :command:`convert` command is used by default.

Reviewer: ```suggestion
   On Unix platforms, the command :command:`convert` is used by default.
```

>    On Windows platform, :command:`magick` command is used by default.

Reviewer: ```suggestion
   On Windows, the command :command:`magick` is used by default.
```

   Additional command-line arguments to give to :command:`convert`, as a list.
   The default is an empty list ``[]``.

>    On Windows platform, it defaults to ``["convert"]``.

Reviewer: ```suggestion
   On Windows, it defaults to ``["convert"]``.
```
